modules/core/src/utils/texture.js

```diff
@@ -28,9 +28,20 @@ export function createTexture(layer, image) {
     image = {data: image};
   }
 
+  let specialTextureParameters = null;
+  if (image.compressed) {
+    specialTextureParameters = {};
+    specialTextureParameters[gl.TEXTURE_MIN_FILTER] =
+      image.data.length > 1 ? gl.LINEAR_MIPMAP_NEAREST : gl.LINEAR;
+  }
```

Comment:
```suggestion
  if (image.compressed) {
    specialTextureParameters = {};
    specialTextureParameters[gl.TEXTURE_MIN_FILTER] =
      image.data.length > 1 ? gl.LINEAR_MIPMAP_NEAREST : gl.LINEAR;
  }
```

Reply:
Image data length can be more than 1. This means a multi-level compressed texture. We have to be able to show such images as well.

Reply:
Sorry, I thought it was the same as the default. Corrected.